wikimedia/wikipedia-ios
# Bug fix - Native Editor template syntax highlights

**Phabricator:** 
https://phabricator.wikimedia.org/T348074

### Notes
This fixes a couple of issues I noticed on template syntax highlighting.

### Test Steps
1. Go to article Hurricane Erika (2003) on EN Wikipedia.
2. Confirm lines in infobox template have correct syntax highlighting.
3. Go to article Ford Island on EN Wikipedia.
4. Confirm pronunciation in intro text template has correct syntax highlighting.

### Screenshots/Videos
Before:
![IMG_0293](https://github.com/wikimedia/wikipedia-ios/assets/3620196/a99db27f-ca55-4aac-965c-9b41ee4fd1dc)
![IMG_0292](https://github.com/wikimedia/wikipedia-ios/assets/3620196/f9806c67-fbf9-4cab-9bfb-59c6eeac608c)

After:
![IMG_0295](https://github.com/wikimedia/wikipedia-ios/assets/3620196/8f8184a6-149f-46bc-b84e-34064cbcfe8e)
![IMG_0294](https://github.com/wikimedia/wikipedia-ios/assets/3620196/ea383f4c-e1dc-413d-b5a5-2c88c65d09c6)
